packages/tools/examples/videoTools/index.ts

```diff
@@ -92,74 +117,357 @@ addButtonToToolbar({
   },
 });
 
+const toolsNames = [
+  LengthTool.toolName,
+  ProbeTool.toolName,
+  RectangleROITool.toolName,
+  EllipticalROITool.toolName,
+  CircleROITool.toolName,
+  BidirectionalTool.toolName,
+  AngleTool.toolName,
+  CobbAngleTool.toolName,
+  ArrowAnnotateTool.toolName,
+  PlanarFreehandROITool.toolName,
+  VideoRedactionTool.toolName,
+];
+let selectedToolName = toolsNames[0];
+
+addDropdownToToolbar({
+  options: { values: toolsNames, defaultValue: selectedToolName },
+  onSelectedValueChange: (newSelectedToolNameAsStringOrNumber) => {
+    const newSelectedToolName = String(newSelectedToolNameAsStringOrNumber);
+    const toolGroup = ToolGroupManager.getToolGroup(toolGroupId);
+
+    // Set the new tool active
+    toolGroup.setToolActive(newSelectedToolName, {
+      bindings: [
+        {
+          mouseButton: MouseBindings.Primary, // Left Click
+        },
+      ],
+    });
+
+    // Set the old tool passive
+    toolGroup.setToolPassive(selectedToolName);
+
+    selectedToolName = <string>newSelectedToolName;
+  },
+});
+
 addButtonToToolbar({
-  id: 'previous',
-  title: 'previous',
+  id: 'Next',
+  title: 'Next',
   onClick() {
-    viewport.scroll(-1);
+    navigateRelative(1);
   },
 });
 
 addButtonToToolbar({
-  id: 'next',
-  title: 'next',
+  id: 'Previous',
+  title: 'Previous',
   onClick() {
-    viewport.scroll(1);
+    navigateRelative(-1);
   },
 });
 
 addButtonToToolbar({
-  id: 'jump',
-  title: 'jump to 50',
+  id: 'Clear',
+  title: 'Clear Range',
   onClick() {
-    viewport.setTime(50);
+    viewport.setRange(null);
+    viewport.play();
   },
 });
 
-const playbackSpeeds = [
-  '0',
-  '0.075',
-  '0.15',
-  '0.25',
-  '0.5',
-  '0.75',
-  '1',
-  '2',
-  '3',
-  '4',
-  '10',
-];
-
-const toolbar = document.getElementById('demo-toolbar');
-const rateTitle = document.createElement('div');
-rateTitle.style.display = 'inline';
-rateTitle.innerText = 'Playback Rate:';
-toolbar.appendChild(rateTitle);
-addDropdownToToolbar({
-  options: { values: playbackSpeeds, defaultValue: '1', id: 'frameRate' },
-  onSelectedValueChange: (newSelectedToolNameAsStringOrNumber) => {
-    const newPlaybackSpeed = Number(newSelectedToolNameAsStringOrNumber);
-    viewport.setPlaybackRate(newPlaybackSpeed);
+addButtonToToolbar({
+  id: 'AddSelection',
+  title: 'Add',
+  onClick() {
+    viewport.pause();
+    if (!selectedAnnotation.annotationUID) {
+      return;
+    }
+    const newSelection = setSelection(
+      selectedAnnotation.annotationUID,
+      null,
+      viewport.getFrame()
+    );
+    updateAnnotationDiv(selectedAnnotation.annotationUID, newSelection);
+    fireUpdateEvent();
   },
 });
 
```

Comment:
I'm not sure why we need Add Selection, since they should be able to just click and select right?